internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,57 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// Разделяем строку на слайс строк
+	parts := strings.Split(data, ",")
+
+	// Проверяем, что длина слайса равна 2
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("ошибка: ожидается два значения, получено %d", len(parts))
+	}
+
+	// Парсим количество шагов
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("ошибка при парсинге шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("ошибка при парсинге шагов: %w", err)
```